examples/4_point_correlator.jl

```diff
@@ -37,12 +37,7 @@ function main(; N)
 
   ops = ("Cdagup", "Cdagdn", "Cdn", "Cup")
 
-  sites = NTuple{4,Int}[]
-  for a in 1:(N - 3)
-    for b in (a + 2):(N - 1)
-      push!(sites, (a, a + 1, b, b + 1))
-    end
-  end
+  sites = vec([tuple(i, j, k, l) for i in 1:N, j in 1:N, k in 1:N, l in 1:N])
```

Comment:
```suggestion
  sites = vec([(i, j, k, l) for i in 1:N, j in 1:N, k in 1:N, l in 1:N])
```
`(1, 2)` is a shorthand for `tuple(1, 2)`.